bin/resin-preload

```diff
@@ -88,6 +92,21 @@ if (argv.includes('--version') || argv.includes('-v')) {
   process.exit(0)
 }
 
+const getResinSdk = (options) => {
+  // Creates a temporary directory for resin sdk so it won't replace any existing token.
+  const resinSdkOptions = { apiKey: options.apiKey, apiUrl: options.apiHost }
+  return tmp.dirAsync({ dir: os.tmpdir(), unsafeCleanup: true })
```

Comment:
Doesn't break anything, but I don't think you need the `dir` argument here - `os.tmpdir()` is the default.